### PR TITLE
New module stock_reserve_mrp

### PR DESCRIPTION
**Pull request by ghost**
This module allows manufacturing orders to reserve stock.

### CODE REVIEW

File: stock_reserve_mrp/README.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Stock Reserve MRP
+=================
+
+This module allows an automatic reserve when the product is finished.
```
Comment (elicoidal) [inline stock_reserve_mrp/README.rst]: Thanks for your contribution.
s/reserve/reservation

File: stock_reserve_mrp/README.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+
+Credits
```
Comment (elicoidal) [inline stock_reserve_mrp/README.rst:17]: I miss bug tracker section

File: stock_reserve_mrp/__init__.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline stock_reserve_mrp/__init__.py:1]: Please use simple headers following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: stock_reserve_mrp/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Stock Reserve MRP',
+    'version': '1.0',
+    'category': 'Manufacturing',
+    'sequence': 10,
+    'summary': 'This module adds an additional status to handle '
```
Comment (elicoidal) [inline stock_reserve_mrp/__openerp__.py]: This description does not fit the object of the module

File: stock_reserve_mrp/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    'in production orders',
+    'author': 'ClearCorp',
+    'website': 'http://clearcorp.co.cr',
+    'depends': ['mrp', 'stock_reserve'],
+    'data': ['stock_reserve_mrp_view.xml'],
+    'test': [],
```
Comment (elicoidal) [inline stock_reserve_mrp/__openerp__.py]: remove empty keys

File: stock_reserve_mrp/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Stock Reserve MRP',
+    'version': '1.0',
```
Comment (elicoidal) [inline stock_reserve_mrp/__openerp__.py]: Please follow the guidelines from:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
Version 8.0.1.0.0?

File: stock_reserve_mrp/mrp.py [hunk 7]
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# © 2015 ClearCorp S.A.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
+
+
+class mrp_bom(models.Model):
```
Comment (gurneyalex) [inline stock_reserve_mrp/mrp.py:8]: in stock.py you use camel case for the class name which is the recommended practise. Can you rename this class to MrpBom?

File: stock_reserve_mrp/mrp.py [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+        _inherit = "mrp.bom"
+
+        reserve_stock = fields.Boolean('Reserve Finished Goods', default=True)
+
+
+class mrp_production(models.Model):
```
Comment (gurneyalex) [inline stock_reserve_mrp/mrp.py:14]: MrpProduction

File: stock_reserve_mrp/mrp.py [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+    @api.multi
+    def product_id_change(self, product_id, product_qty=0):
+        res = super(mrp_production, self).product_id_change(
+            product_id=product_id, product_qty=product_qty)
+        if self.bom_id.reserve_stock:
+            res['value']['reserve_stock'] = self.bom_id.reserve_stock
+        return res
+
+    def action_produce(self, cr, uid, production_id, production_qty,
+                       production_mode, wiz=False, context=None):
```
Comment (gurneyalex) [inline stock_reserve_mrp/mrp.py:28]: is it possible to port this method to the new API?

**Comment by gurneyalex** [comment]
In addition to the notes above, could you:
- move the stock.py and mrp.py files to a models/ subpackage
- move the views to a views/ subdirectory
- add an automated test

Thanks for your time

**Comment by gurneyalex** [comment]
user lost interes, no CLA, old PR -> closing

**Pull request closed without merging**